stable/prometheus-operator/values.yaml

```diff
@@ -1864,6 +1864,14 @@ prometheus:
     #   - regex: (kubernetes_io_hostname|failure_domain_beta_kubernetes_io_region|beta_kubernetes_io_os|beta_kubernetes_io_arch|beta_kubernetes_io_instance_type|failure_domain_beta_kubernetes_io_zone)
     #     action: labeldrop
 
+    ## If additional scrape configurations are already deployed in a single secret file you can use this section.
+    ## Expected values are the cm name and key
+    ## Cannot be used with additionalScrapeConfigs
+    additionalScrapeConfigsSecret: {}
+      # enabled: false
+      # SecretName:
```

Comment:
By convention these should not be capitalised. 
/hold cancel
/ok-to-test

Reply:
you mean SecretName should be secret name and same for SecretKey?